README.adoc

```diff
@@ -68,41 +53,54 @@ NOTE: This file is where we alias the `latest` path in URLs.
 # Redirect latest imdg alias to the latest version
 /imdg/latest/*  /imdg/4.1.2/:splat 200!
 ----
-
-The documentation site will now be built, including content from your tagged version.
++
+- In the `search-config.json` file, change the value of the `variables.version` field to your release version.
++
+NOTE: This file is where we specify which documentation version to index for the search.
++
+[source,json]
+----
+{
+  "index_name": "prod_hazelcast_docs",
+  "start_urls": [
+    {
+      "url": "https://docs.hazelcast.com/imdg/(?P<version>.*?)/",
+      "tags": [
+        "imdg"
+      ],
+      "variables": {
+        "version": ["4.2"]
+      },
+----
 
 == Major and Minor Releases
 
 This section guides you through the steps for releasing documentation for major and minor releases.
 
-. If you're writing content for a new major or minor release, make your changes in the `master` branch.
+. Create a new temporary branch from the `master` branch. For example if you’re working on content for version 5.0, create a branch named `5.0`. You can add the new content and/or fixes to this branch before the release.
 
-. If you're writing content for a new minor version of a previous major release, create a `\{version\}.y` branch and make your changes in that branch. For example if version 5.0 was already released and you plan to add content for version 4.3 then you need to create a branch named`4.y`.
+. In the `antora.yml` file of your temporary branch, do the following:
 +
-TIP: For guidance on writing content, see the {url-contribute}[contribution guidelines].
-
-. When you are ready to release, create a new release branch from the `master` or `.y` branch and name it with the full version number that you are releasing. For example, if you are releasing version 5.0, name the branch 5.0.
-
-. If you are releasing a new `latest-dev` version, in the `antora.yml` file of the `master` branch, increment the `version` field to the `latest-dev` version. For example, `5.0-SNAPSHOT` becomes `5.1.0-SNAPSHOT`.
-
-. To make it easier to amend your release for future patch versions or to fix typos,  create a `\{version\}.z` branch from your release branch. For example if your release branch is named `5.0`, create a branch named `5.0.z`.
-
-. In the `antora.yml` file of the `.z` branch, increment the `version` field. For example, `5.0` becomes `5.0.1-SNAPSHOT`.
-
-. In the `antora.yml` file of your release branch, update the `version` field with the version of IMDG that you are working on, and delete the `prerelease` field. For example version `5.0-SNAPSHOT` becomes `5.0`.
+- Update the `version`, `display_version`, and `asciidoc.attributes.full-version` fields with the major and minor version of IMDG that you are releasing.
```

Comment:
What's actually the difference between `version` and `display_version`? When they include different values?

Reply:
`version` is included as a path in the URL. `display_version` is displayed in the version selector dropdown.

I've used both just in case we ever want to use it in the future.

Also, we will use `asciidoc.attributes.full-version` as variables in the docs for things like download links. For path releases, this variable will contain the full version (major.minor.patch)